app/liquid_tags/details_tag.rb

```diff
@@ -0,0 +1,27 @@
+class DetailsTag < Liquid::Block
+  include ActionView::Helpers::SanitizeHelper
+
+  PARTIAL = "liquids/details".freeze
+
+  def initialize(_tag_name, summary, _parse_context)
+    super
+    @summary = sanitize(summary.strip)
+  end
+
+  def render(_context)
+    content = Nokogiri::HTML.parse(super)
+    parsed_content = sanitize(content.xpath("//html/body").inner_html)
```

Comment:
I don't _think_ this is a security issue to use `inner_html`, but please let me know if I missed something here!

Reply:
Maybe @justinlarson can weigh in on this?

Reply:
It won't be because when you throw it into `ActionController::Base.new.render_to_string`, the content will be escaped